pkg/internalmetrics/metrics.go

```diff
@@ -0,0 +1,46 @@
+package internalmetrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+// These constants define the different label names for the different metric timeseries
+const (
+	labelVersion      = "version"
+	labelNamespace    = "namespace"
+	labelSMCPVersion  = "smcp_version"
+	labelSMCPTopology = "smcp_topology"
```

Comment:
Can we rename this to "mode"? In 2.4, `controlPlaneMode` will be renamed to `mode`, so it makes sense to use the same term in metrics. Is there a reason we're calling this "topology"?

Reply:
Right I changed it to `smcp_mode` instead